frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -1,153 +1,251 @@
+                Basic Information
+              </h2>
+              <div className="space-y-4">
                 <div>
-                  <label className="block text-gray-700 mb-2">Bot Name</label>
-                  <input 
-                    type="text" 
-                    placeholder="Enter bot name" 
-                    className="w-full border border-gray-300 rounded-lg px-4 py-3 shadow-sm focus:outline-none focus:ring-2 focus:ring-blue-500 transition"
+                  <label className="block text-gray-700 font-medium mb-2">
+                    Bot Name
+                  </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)